pkg/sdn/plugin/node.go

```diff
@@ -220,6 +220,11 @@ func (node *OsdnNode) Start() error {
 		return err
 	}
 
+	// podManager must be created early because other goroutines
+	// may call into it before its started due to event watches
+	log.V(5).Infof("Creating openshift-sdn pod manager")
+	node.podManager = newPodManager(node.kClient, node.policy, node.mtu, node.oc)
```

Comment:
"it's" (in the comment)

should this be in NewNodePlugin() rather than Start()

also, maybe move `node.localSubnetCIDR, err = node.getLocalSubnet()` to before the node_iptables calls, and then add a comment between the two indicating that that's the point after which all of node's fields have been initialized?